bin/compile

```diff
@@ -480,6 +501,9 @@ setupProcfile() {
 
 case "${TOOL}" in
     gomodules)
+        ensureBZR "${BazaarVersion}"
+        ensureHG "${MercurialVersion}"
```

Comment:
Was this not running before?

Reply:
hi!

Maybe that was the case but I don't remember